exporter/src/main/as/flump/export/texturepacker/MaxRectMultiPacker.as

```diff
@@ -0,0 +1,105 @@
+package flump.export.texturepacker {
```

Comment:
The space algorithm that tries to use fewest number of textures:  
1. Sort images by area and size. 
1. Find the biggest image, make an atlas that has the size of the next power of 2.
2. Try to pack every texture into it.
3. if one does not fit:
   If the atlas size is < 2048 grow the atlas to 2x of its current size and repeat from 2) 
   else create a new atlas and repeat from 1)